src/node.rs

```diff
@@ -424,6 +424,11 @@ impl Node {
         self.chain().into_iter().flat_map(Chain::elders)
     }
 
+    /// Returns the members in our section and elders we know.
+    pub fn known_nodes(&self) -> impl Iterator<Item = &P2pNode> {
+        self.chain().into_iter().flat_map(Chain::our_joined_members)
```

Comment:
Shouldn't this return `Chain::known_nodes` ?

Reply:
Yes, I messed something up with the review fixes and tests are failing as well